Structures.cpp

```diff
@@ -0,0 +1,111 @@
+#include <iostream>
+
+const int SUBJECT_NUM = 4;
+const int CLASS_SIZE = 5;
+
+struct Student {
+    const char* name;
+    int marks[SUBJECT_NUM];
+};
+
+double averageMark(Student student);
```

Comment:
const Student &
to not create additional copy

Reply:
Done in last commit